src/transforms/remap.rs

```diff
@@ -129,6 +130,7 @@ pub struct RemapConfig {
     /// to a specially-named output, `dropped`. The original event is annotated with additional
     /// fields describing why the event was dropped.
     #[serde(default = "crate::serde::default_false")]
+    #[configurable(metadata(docs::human_name = "Drop on Abort"))]
```

Comment:
Looks like this should be applied to `drop_on_abort` instead?

Reply:
Fixed.